src/ServiceControl.Persistence.SqlServer/SqlDbCustomCheckDataStore.cs

```diff
@@ -115,39 +115,38 @@ public async Task<QueryResult<IList<CustomCheck>>> GetStats(PagingInfo paging, s
             return new QueryResult<IList<CustomCheck>>(checks, new QueryStatsInfo(null, totalCount));
         }
 
-        public static async Task Setup(string connectionString)
-        {
-            using (var connection = new SqlConnection(connectionString))
-            {
-                var catalog = new SqlConnectionStringBuilder(connectionString).InitialCatalog;
-
-                var createCommand = $@"
-                    IF NOT EXISTS (
-                         SELECT *
-                         FROM {catalog}.sys.objects
-                         WHERE object_id = OBJECT_ID(N'CustomChecks') AND type in (N'U')
-                       )
-                       BEGIN
-                           CREATE TABLE [dbo].[CustomChecks](
-                               [Id] [uniqueidentifier] NOT NULL,
-                               [CustomCheckId] nvarchar(300) NOT NULL,
-                               [Category] nvarchar(300) NULL,
-                               [Status] int NOT NULL,
-                               [ReportedAt] datetime NOT NULL,
-                               [FailureReason] nvarchar(300) NULL,
-                               [OriginatingEndpointName] nvarchar(300) NOT NULL,
-                               [OriginatingEndpointHostId] [uniqueidentifier] NOT NULL,
-                               [OriginatingEndpointHost] nvarchar(300) NOT NULL
-                           ) ON [PRIMARY]
-                       END";
-
-                connection.Open();
-
-                await connection.ExecuteAsync(createCommand).ConfigureAwait(false);
-            }
-        }
+        //public static async Task Setup(string connectionString)
```

Comment:
Should this be commented?

Reply:
It's temporary until I figure out how to best do the setup